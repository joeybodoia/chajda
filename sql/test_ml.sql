CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."');
