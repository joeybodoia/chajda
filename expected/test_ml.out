CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                                  spacy_lemmatize                                                  
-------------------------------------------------------------------------------------------------------------------
   ടെസ്റ്റ്:2 കേസുകൾ:3 സൃഷ്ടിക്കുന്നതിനായി:4 ഞാൻ:5 google:6 വിവർത്തനത്തിലേക്ക്:7 ഇടുന്ന:8 എന്റെ:9 ഉദാഹരണ:10 പരീക്ഷണ:11 വാക്യമാണിത്:12
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                                     spacy_lemmatize                                      
------------------------------------------------------------------------------------------
   ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                  spacy_lemmatize                                                  
-------------------------------------------------------------------------------------------------------------------
   ടെസ്റ്റ്:2 കേസുകൾ:3 സൃഷ്ടിക്കുന്നതിനായി:4 ഞാൻ:5 google:6 വിവർത്തനത്തിലേക്ക്:7 ഇടുന്ന:8 എന്റെ:9 ഉദാഹരണ:10 പരീക്ഷണ:11 വാക്യമാണിത്:12
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                     spacy_lemmatize                                      
------------------------------------------------------------------------------------------
   ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                       spacy_lemmatize                                                        
------------------------------------------------------------------------------------------------------------------------------
   ":2 ടെസ്റ്റ്:3 കേസുകൾ:4 സൃഷ്ടിക്കുന്നതിനായി:5 ഞാൻ:6 google:7 വിവർത്തനത്തിലേക്ക്:8 ഇടുന്ന:9 എന്റെ:10 ഉദാഹരണ:11 പരീക്ഷണ:12 വാക്യമാണിത്.:13 ":14
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                        spacy_lemmatize                                        
-----------------------------------------------------------------------------------------------
   " ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്. "
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                       spacy_lemmatize                                                        
------------------------------------------------------------------------------------------------------------------------------
   ":2 ടെസ്റ്റ്:3 കേസുകൾ:4 സൃഷ്ടിക്കുന്നതിനായി:5 ഞാൻ:6 google:7 വിവർത്തനത്തിലേക്ക്:8 ഇടുന്ന:9 എന്റെ:10 ഉദാഹരണ:11 പരീക്ഷണ:12 വാക്യമാണിത്.:13 ":14
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                        spacy_lemmatize                                        
-----------------------------------------------------------------------------------------------
   " ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്. "
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                                  spacy_lemmatize                                                  
-------------------------------------------------------------------------------------------------------------------
   ടെസ്റ്റ്:2 കേസുകൾ:3 സൃഷ്ടിക്കുന്നതിനായി:4 ഞാൻ:5 Google:6 വിവർത്തനത്തിലേക്ക്:7 ഇടുന്ന:8 എന്റെ:9 ഉദാഹരണ:10 പരീക്ഷണ:11 വാക്യമാണിത്:12
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                                     spacy_lemmatize                                      
------------------------------------------------------------------------------------------
   ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                  spacy_lemmatize                                                  
-------------------------------------------------------------------------------------------------------------------
   ടെസ്റ്റ്:2 കേസുകൾ:3 സൃഷ്ടിക്കുന്നതിനായി:4 ഞാൻ:5 Google:6 വിവർത്തനത്തിലേക്ക്:7 ഇടുന്ന:8 എന്റെ:9 ഉദാഹരണ:10 പരീക്ഷണ:11 വാക്യമാണിത്:12
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                     spacy_lemmatize                                      
------------------------------------------------------------------------------------------
   ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                       spacy_lemmatize                                                        
------------------------------------------------------------------------------------------------------------------------------
   ":2 ടെസ്റ്റ്:3 കേസുകൾ:4 സൃഷ്ടിക്കുന്നതിനായി:5 ഞാൻ:6 Google:7 വിവർത്തനത്തിലേക്ക്:8 ഇടുന്ന:9 എന്റെ:10 ഉദാഹരണ:11 പരീക്ഷണ:12 വാക്യമാണിത്.:13 ":14
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                        spacy_lemmatize                                        
-----------------------------------------------------------------------------------------------
   " ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്. "
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                       spacy_lemmatize                                                        
------------------------------------------------------------------------------------------------------------------------------
   ":2 ടെസ്റ്റ്:3 കേസുകൾ:4 സൃഷ്ടിക്കുന്നതിനായി:5 ഞാൻ:6 Google:7 വിവർത്തനത്തിലേക്ക്:8 ഇടുന്ന:9 എന്റെ:10 ഉദാഹരണ:11 പരീക്ഷണ:12 വാക്യമാണിത്.:13 ":14
(1 row)

SELECT spacy_lemmatize('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                        spacy_lemmatize                                        
-----------------------------------------------------------------------------------------------
   " ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്. "
(1 row)

SELECT spacy_tsvector('ml',' "ടെസ്റ്റ് കേസുകൾ സൃഷ്ടിക്കുന്നതിനായി ഞാൻ Google വിവർത്തനത്തിലേക്ക് ഇടുന്ന എന്റെ ഉദാഹരണ പരീക്ഷണ വാക്യമാണിത്."');
                                                            spacy_tsvector                                                             
---------------------------------------------------------------------------------------------------------------------------------------
 'google':6 'ഇടുന്ന':8 'ഉദാഹരണ':10 'എന്റെ':9 'കേസുകൾ':3 'ഞാൻ':5 'ടെസ്റ്റ്':2 'പരീക്ഷണ':11 'വാക്യമാണിത്':12 'വിവർത്തനത്തിലേക്ക്':7 'സൃഷ്ടിക്കുന്നതിനായി':4
(1 row)

